app/src/androidTest/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordTest.kt

```diff
@@ -0,0 +1,126 @@
+    val facilityId = UUID.fromString("f12c2316-85e4-466e-9c92-efa52605c6b9")
+    val requesterId = UUID.fromString("7a131902-c214-4aa9-800f-201513d77fd7")
+    val date = LocalDate.parse("2020-09-03")
+    val prescribedDrugUuid = UUID.fromString("a6e44fdd-d029-47b0-9964-2ad4747166ec")
+    val prescribedDrugUuid2 = UUID.fromString("4904369e-c425-4098-bde1-560e09eb4383")
+
+    val teleconsultRecordInfo = TestData.teleconsultRecordInfo(
+        recordedAt = Instant.parse("2020-09-03T00:00:00Z"),
+        teleconsultationType = TeleconsultationType.Audio,
+        patientTookMedicines = Answer.Yes,
+        patientConsented = Answer.No,
+        medicalOfficerNumber = "22222222"
+    )
+
+    val teleconsultRequestInfo = TestData.teleconsultRequestInfo(
+        requesterId = requesterId,
+        facilityId = facilityId,
+        requestedAt = Instant.parse("2020-09-02T00:00:00Z"),
+    )
+
+    val teleconsultRecord1 = TestData.teleconsultationRecordInfo(
+        id = teleconsultRecordId1,
+        patientId = patientId,
+        medicalOfficerId = medicalOffcerId,
+        teleconsultRecordInfo = teleconsultRecordInfo,
+        teleconsultRequestInfo = teleconsultRequestInfo,
+        timestamps = Timestamps(
+            createdAt = date.toUtcInstant(userClock),
+            updatedAt = date.toUtcInstant(userClock),
+            deletedAt = null
+        )
+    )
+
+    val teleconsultRecord2 = TestData.teleconsultationRecordInfo(
+        id = teleconsultRecordId2,
+        patientId = patientId,
+        medicalOfficerId = medicalOffcerId,
+        teleconsultRecordInfo = teleconsultRecordInfo,
+        teleconsultRequestInfo = teleconsultRequestInfo,
+        timestamps = Timestamps(
+            createdAt = date.toUtcInstant(userClock),
+            updatedAt = date.toUtcInstant(userClock),
+            deletedAt = null
+        )
+    )
+
+    val teleconsultRecordPrescribedDrug1 = TestData.teleconsultationRecordWithPrescribedDrugs(
+        teleconsultRecordId = teleconsultRecordId1,
+        prescribedDrugUuid = prescribedDrugUuid,
+    )
+
+    val teleconsultRecordPrescribedDrug2 = TestData.teleconsultationRecordWithPrescribedDrugs(
+        teleconsultRecordId = teleconsultRecordId2,
+        prescribedDrugUuid = prescribedDrugUuid2,
+    )
+
+    teleconsultRecordDao.save(
+        listOf(
+            teleconsultRecord1,
+            teleconsultRecord2
+        )
+    )
+
+    val expectedTeleconsultRecordPrescribedDrugList = listOf(
+        teleconsultRecordPrescribedDrug1,
+        teleconsultRecordPrescribedDrug2
+    )
+
+    teleconsultRecordPrescribedDrugDao.save(
+        expectedTeleconsultRecordPrescribedDrugList
+    )
+
+    // when
+    val teleconsultRecordPrescribedDrugList = teleconsultRecordPrescribedDrugDao
+        .getAll()
+
+    // then
+    assertThat(teleconsultRecordPrescribedDrugList).isEqualTo(expectedTeleconsultRecordPrescribedDrugList)
```

Comment:
This does not test the complete picture since we are not verifying whether multiple prescribed drugs for a single teleconsult record are being saved correctly or not. We should just save a single complete teleconsult record with multiple prescribed drug UUIDs and then verify that loading that `TeleconsultRecordWithPrescribedDrugs` record is as expected.